core/src/main/java/com/linecorp/armeria/common/RedirectType.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+/**
+ * RedirectType for simply redirection in {@link HttpResponse}.
+ */
+public final class RedirectType {
```

Comment:
How about removing this and using `HttpStatus` instead? We could add the same validation we did in: https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/RedirectService.java#L244-L247

Reply:
OK. I fixed this PR like `RedirectService` using `HttpStatus`. And refactored reusing `ofRedirect` method.

Reply:
That was fast! Thank you, @cnabro 🙇